packages/synapse-react-client/src/components/DataGrid/StartGridSession.tsx

```diff
@@ -39,6 +40,13 @@ export const StartGridSession = forwardRef<
       onSessionChange(session)
       onReplicaChange(replica.replicaId!)
     },
+    onError(error) {
+      const message =
+        error && typeof error === 'object' && 'reason' in error
+          ? (error.reason as string)
+          : 'An error occurred'
+      displayToast(message, 'danger')
```

Comment:
Since this could be any error, use the `message` property instead of `reason` (which is only on our `SynapseClientError` type). SynapseClientError puts the `reason` property in `message`, so you won't need to do anything special for it
```suggestion
      const message =
        error && typeof error === 'object' && 'message' in error && typeof error.message === 'string'
          ? (error.message as string)
          : 'An error occurred'
      displayToast(message, 'danger')
```